### PR TITLE
WIP: filesystem to mongo

### PR DESCRIPTION
**Pull request by st3107**
Add functionality: transfer the database from fielsystem to mongodb

### CODE REVIEW

**Comment by st3107** [comment]
@sbillinge Ready for review. I am not sure why the codecov thought the coverage decreased. I add the test for the fs-to-mongo. This should cover almost all the code I added.

I cannot see the report of codecov so that I cannot know where to improve. I am not sure if someone could authorize me of the codecov.

File: regolith/commands.py [hunk 1]
```diff
@@ -180,6 +185,23 @@ def yaml_to_json(rc):
         fsclient.yaml_to_json(inp, out)
 
 
+def fs_to_mongo(rc: RunControl) -> None:
+    """Import database from filesystem to mongo db.
```
Comment (sbillinge) [inline regolith/commands.py:189]: I wonder if this is clearer if it says "Convert database collection from filesystem to mongo db" rather than import

**Comment by st3107** [comment]
Done. Ready to merge.

**Comment by sbillinge** [comment]
@scopatz @CJ-Wright This is some basic infrastructure for regolith so I would rather that you look over it quickly.  We are playing around with moving some of the databases to a mongo backend and Songsheng (@st3107 ) has written some code that should help with that.  I am happy to merge it, but would love for either or both of you to look it over quickly and make sure it is ok, or make suggestions.   Thanks

**Comment by sbillinge** [comment]
@CJ-Wright @scopatz any thoughts?

**Comment by st3107** [comment]
I need to do some further enhancement. There is a bug when trying to import database to mongod atlas

**Comment by st3107** [comment]
Done. Ready for review.

**Comment by sbillinge** [comment]
thanks @st3107 

I think that tools is causing coverage to fail.  Maybe we need a  quick hackathon to get more coverage in tools....

**Comment by st3107** [comment]
I am not sure about the mechanism in calculation of coverage. I can make sure all the functions I add have been tested.

**Comment by sbillinge** [comment]
yes on tests for functions, but we need to add tests for some functions that got merged before without tests.

I think one test is that coverage  >65% or so and another test on the change in coverage.

**Pull request merged**